moped-editor/src/views/projects/newProjectView/NewProjectMap.js

```diff
@@ -205,6 +205,8 @@ const handleSelectedFeatureUpdate = (
     const bboxLine = polygonToLine(bboxPolygon(bbox));
     const intersectsWithBounds = booleanIntersects(bboxLine, selectedFeature);
 
+    // queryCtnFeatureService queries on "where: `PROJECT_EXTENT_ID=${projectExtentId}`", Should this be
+    // updated to check for CTN_SEGMENT_ID?
```

Comment:
im not sure how to test that fetching the complete geometry from agol is working, I can't tell if this should be updated or left as is, @johnclary ?

Reply:
ah yes, good question—added this todos in [my branch](https://github.com/cityofaustin/atd-moped/pull/553) from yours

Reply:
AGOL was not working, and we weren't catching the API errors (they were return status code `200` :/ 

Fixed in my branch:
https://github.com/cityofaustin/atd-moped/pull/553/commits/760e85dbf1822e510f71ec574e1d652fa504a919

Reply:
btw, the trick to testing this is to enter component edit mode, then zoom in close on a looong street segment. when you click it, `queryCtnFeatureService` will be invoked.

<img width="834" alt="Screen Shot 2022-02-05 at 7 27 10 PM" src="https://user-images.githubusercontent.com/14793120/152664581-bc9c6ebe-c73c-4d30-8595-7c8918517d73.png">

Reply:
AGOL returns a 200 and then nests an error with code 400 in that? what a bummer

Reply:
it's baffling!

Reply:
it's baffling!